mne/conftest.py

```diff
@@ -140,6 +140,10 @@ def pytest_configure(config):
     ignore:Implementing implicit namespace packages.*:DeprecationWarning
     ignore:Deprecated call to `pkg_resources.*:DeprecationWarning
     ignore:pkg_resources is deprecated as an API.*:DeprecationWarning
+    # h5py
+    ignore:`product` is deprecated as of NumPy.*:DeprecationWarning
+    # TODO: This one we should fix with .item()s
+    ignore:Conversion of an array with ndim > 0 to a scalar.*:DeprecationWarning
```

Comment:
Still a bunch to fix @drammock, but I'll get them tomorrow. For now I'll get this in and fix our CI requirements...